### DISCUSSION TITLE
Use BEVFusion for camera only

### DISCUSSION THREAD

**Discussion by flstahl** [open-mmlab/mmdetection3d#2888] [Q&A]
Hi folks,
I am currently experimenting with the BEVFusion model from projects/BEVFusion. There are a couple of different modalities how this model can be used. There are commands for lidar-only training and lidar-camera training. I am wondering if and how you could do camera-only?

I saw that in the original paper/repo from MIT Han Lab (https://github.com/mit-han-lab/bevfusion) they also have camera-only metrics. However, I have issues getting their repo to run on my Nvidia H100 GPU which is why I would like to stick to this repo here.

Can anyone help me out?

Thanks a lot!

**Comment by Wangquans** [comment]
I also encountered this problem. It can be achieved theoretically by modifying the configuration file and code, but it was too difficult and I gave up in the end. If this problem is solved, please let me know. Thank you
